magento/adobe-stock-integration
# Display image licensing confirmation popup

### Description (*)
This PR provides functionality for showing license confirmation popup with information about the current license quota retrieved from Adobe Stock API

### Fixed Issues (if relevant)

1. #289 : Display image licensing confirmation popup

### Manual testing scenarios (*)
Preconditions: Adobe stock integration package installed and API keys configured

1. Go to Admin -> Content -> Pages -> Choose a page | create new
2. Open dialog for inserting image to a content
3. Click on "Open Adobe Stock"
4. Choose an image 
5. Click on "License and Save" button

If everything goes well, you should see the confirmation popup with the quota information
